test/create_test.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
```

Comment:
Please remove this file. I'll write a test regarding the multi app creation scenario later.

Reply:
Okay, removed.